spec/requests/admin/badge_achievements_spec.rb

```diff
@@ -104,7 +104,6 @@
       expect do
         delete admin_badge_achievement_path(badge_achievement.id)
       end.to change { BadgeAchievement.all.count }.by(-1)
-      expect(response.body).to redirect_to admin_badge_achievements_path
```

Comment:
I removed this part of the spec, seeing as the Badge_Achievement Removal process is now async. Redirect is now unnecessary.